cephfs/cephfs.go

```diff
@@ -7,17 +7,26 @@ package cephfs
 #include <cephfs/libcephfs.h>
 */
 import "C"
-import "fmt"
-import "unsafe"
 
-//
+import (
+	"fmt"
+	log "github.com/sirupsen/logrus"
```

Comment:
is this used?

Reply:
"fmt" is used by `func (e CephError) Error() string {` to format the Error string.

and log is used by every `log.Errorf` statement.

Logrus was already used (i presume because I saw it get pulled in the dockerfile).

Reply:
ah yes @isi-lincoln  thx i just overlooked it :+1: